store-implementation/federated-store/src/test/java/uk/gov/gchq/gaffer/federatedstore/MergeElementFunctionAggregationViewFilterTest.java

```diff
@@ -110,17 +113,14 @@ public void shouldOnlyReturn1EntitySmallerThanViewFilter() throws Exception {
         //then
         assertThat(elementsWithPropertyLessThan2)
                 .isNotNull()
-                .withFailMessage("should return entity with property 1 which is less than view filter 2")
-                .contains(entity1)
                 .withFailMessage("should not return entity with property 99 which is more than view filter 2")
                 .doesNotContain(entity99)
                 .withFailMessage("should contain only 1")
-                .hasSize(1);
```

Comment:
The fail message should also have been removed.

Reply:
done